File: src/main/java/destiny/tools/UrlShorterService.java
/**
 * @author smallufo
 * Created on 2011/2/28 at 下午4:44:12
 */
package destiny.tools;

public interface UrlShorterService {

  /** 提供 UrlShorter 的單一進入點，處理 Exceptions */
  String getShortUrl(String longUrl);

  void resetUrlShorters();
}

File: src/main/java/destiny/tools/UrlShorterService.kt
/**
 * @author smallufo
 * Created on 2011/2/28 at 下午4:44:12
 */
package destiny.tools

interface UrlShorterService {

  /** 提供 UrlShorter 的單一進入點，處理 Exceptions  */
  fun getShortUrl(longUrl: String): String

  fun resetUrlShorters()
}
